components/observatorium.libsonnet

```diff
@@ -118,9 +138,8 @@ local api = (import 'observatorium/observatorium-api.libsonnet');
     } + {
       ['thanos-' + name]: obs.thanos.manifests[name]
       for name in std.objectFields(obs.thanos.manifests)
-    } + if std.length(obs.config.loki) != 0 then {
+    } + {
       ['loki-' + name]: obs.loki.manifests[name]
       for name in std.objectFields(obs.loki.manifests)
-    }
-    else {},
+    },
```

Comment:
Is the `if std.length(obs.config.loki) != 0` still ensured? IIRC, this check was added by ACM folks to deploy observatorium w/o loki.

Reply:
We now encourage an alternative way to build up the configuration in the downstream repo. Rather than inheritance, we want them to compose theirs on setup utilizing functions. So this should be fined.